content/en/docs/getting-started/_index.md

```diff
@@ -221,6 +221,14 @@ oc.exe get all -A --kubeconfig .\kubeconfig
 {{% /tab %}}
 {{< /tabs >}}
 
+
+As a developer, you might want to stop for the day and continue your work later. Stopping and restarting the MicroShift all-in-one container will bring some issues due to the internal IP address change. This is why we recommend developers to use the `pause` and `unpause` Podman/Docker command.
```

Comment:
```suggestion
As a developer, you might want to stop for the day and continue your work later. Stopping and restarting the MicroShift all-in-one container will bring some know issues with the CNI. This is why we recommend developers to use the `pause` and `unpause` Podman/Docker command.
```

Reply:
Just a suggestion, as we are handling now the IP change , but it's more on the ballpark of CNI / CRIO